it doesn't launch otherwise:

Please compile the "tractor.gschema.xml" file.
In GNU/Linux you can copy it from
"/usr/lib/python3.12/site-packages/tractor"
to "/usr/local/share/glib-2.0/schemas" and run:
"sudo glib-compile-schemas /usr/local/share/glib-2.0/schemas"
---
--- a/src/tractor/db.py
+++ b/src/tractor/db.py
@@ -15,7 +15,7 @@ def dconf() -> Gio.Settings:
     """
     schema = "org.tractor"
     schemas = Gio.SettingsSchemaSource.get_default()
-    if not Gio.SettingsSchemaSource.lookup(schemas, schema, False):
+    if not Gio.SettingsSchemaSource.lookup(schemas, schema, True):
         gschema_dir = "/usr/share/glib-2.0/schemas/"
         for directory in GLib.get_system_data_dirs():
             gdir = f"{directory}/glib-2.0/schemas/"
